src/modules/Dashboard/components/ActivitySummaryCard/ActivitySummaryCard.styles.ts

```diff
@@ -0,0 +1,41 @@
+import { styled } from '@mui/material';
+
+import {
+  StyledFlexAllCenter,
+  StyledFlexSpaceBetween,
+  StyledHeadline,
+  theme,
+  variables,
+} from 'shared/styles';
+
+export const StyledContainer = styled(StyledFlexSpaceBetween)`
+  flex-direction: column;
+  padding: ${theme.spacing(2.4)};
+  width: 30.6rem;
+  height: 44rem;
+  border: ${variables.borderWidth.md} solid ${variables.palette.surface_variant};
+  border-radius: ${variables.borderRadius.lg2};
+`;
+
+export const StyledImageContainer = styled(StyledFlexAllCenter)`
+  min-width: 8rem;
+  width: 8rem;
+  height: 8rem;
+  background-color: ${variables.palette.primary_container};
+  border-radius: ${variables.borderRadius.md};
+  overflow: hidden;
+`;
+
+export const StyledImg = styled('img')`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`;
+
+export const StyledActivityName = styled(StyledHeadline)`
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  text-overflow: ellipsis;
+  overflow: hidden;
```

Comment:
Should we do a word break here?

<img width="308" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/6c6cd8e9-538a-4445-a939-de6cfba5d172"> <img width="66" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/c78d5a7c-3441-43ad-a1c3-5ee88a5ae26c"> <img width="308" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/24eb5ba5-b0bb-4465-a8e3-701e81490a84">

```suggestion
  overflow: hidden;
  word-wrap: break-word
```